src/Illuminate/Support/Siesta.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+namespace Illuminate\Support;
+
+use Carbon\CarbonInterval;
+use DateInterval;
+use PHPUnit\Framework\Assert as PHPUnit;
+use RuntimeException;
+
+class Siesta
```

Comment:
I would prefer this class to be named `Pause`, or something similar.

Reply:
I’ve mentioned naming in the description 👍